pom.xml

```diff
@@ -317,7 +317,7 @@
                 </repository>
             </repositories>
         </profile>
-        <profile>
+<!--        <profile>
```

Comment:
You can revert the changes to this file.  The eclipse-sign profile is something for me to fuss with:).